core/regression.py

```diff
@@ -71,7 +75,10 @@ def no_clustering_train(original_test_data, train_data, test_data, regressor):
     y = train_data['remaining_time']
     train_data = train_data.drop('remaining_time', 1)
     regressor.fit(train_data, y)
-    return no_clustering_test(original_test_data, test_data, regressor)
+    split=dict()
```

Comment:
so this is actually a ModelSplit object? If so rename to `model_split` as there is already a Split and that would make it confusing